jaxrs-spec/src/main/asciidoc/chapters/asynchronous_processing/_server_api.adoc

```diff
@@ -29,12 +29,13 @@ Let us illustrate these concepts via an example:
 @Path("/async/longRunning")
 public class MyResource {
 
-    @GET
-    public void longRunningOp(@Suspended final AsyncResponse ar) {
+    @POST
+    public void longRunningOp(final AsyncResponse ar, @Body String message) {
```

Comment:
Dropping `@Suspended` is OK, but why the addition of `@Body` in this example?

Reply:
I changed it to a POST example to show the new `@Body` requirement that I'm documenting below. Is this incorrect? I basically took it from the slideshow.

Reply:
I don't think it is wrong, but seems unnecessary. Note that the same example is repeated later in that chapter, so perhaps it is better to leave it as it was.

Reply:
I think it would be a good idea to have at least one example showing the new `@Body` annotation being used.

That being said, I did miss that they example is repeated later. That example should at least have the `@Suspended` annotation removed regardless.

Reply:
I did remove `@Suspended` from the 2nd example, but I would like some more opinions on changing the example to include `@Body`. I'd be happy to make the 2nd example match the first if that is the consensus.

Reply:
>I think it would be a good idea to have at least one example showing the new @Body annotation being used.
I'd tend to agree since this is where `@Body` is introduced.  Nice to see it in practice.

Reply:
There must be a better example in some other section where we can show the use of `@Body`. This one is really about explaining async.